metricq/types.py

```diff
@@ -48,31 +47,36 @@ class Timedelta:
+        |                  | * :code:`t`: :class:`Timestamp`                  | (the same as :code:`t + d`, see :class:`Timestamp.__add__`)                               |
+        +------------------+--------------------------------------------------+-------------------------------------------------------------------------------------------+
+        | :code:`d1 == d2` | * :code:`d1`, :code:`d2`: :class:`Timedelta`     | :code:`True` if :code:`d1` and :code:`d2` describe the same duration of time              |
+        +------------------+--------------------------------------------------+-------------------------------------------------------------------------------------------+
+        | :code:`d == dt`  | * :code:`d`: :class:`Timedelta`                  | :code:`True` if :code:`d.datetime == dt`                                                  |
+        |                  | * :code:`dt`: :class:`python:datetime.timedelta` | (see :class:`python:datetime.timedelta`)                                                  |
+        +------------------+--------------------------------------------------+-------------------------------------------------------------------------------------------+
+        | :code:`d1 < d2`  | * :code:`d1`, :code:`d2`: :class:`Timedelta`     | :code:`True` if :code:`d1` is shorter than :code:`d2`                                     |
+        +------------------+--------------------------------------------------+-------------------------------------------------------------------------------------------+
+        | :code:`d < dt`   | * :code:`d`: :class:`Timedelta`                  | :code:`True` if :code:`d.datetime < dt`                                                   |
+        |                  | * :code:`dt`: :class:`python:datetime.timedelta` | (see :class:`python:datetime.timedelta`)                                                  |
+        +------------------+--------------------------------------------------+-------------------------------------------------------------------------------------------+
+        | :code:`d * c`    | * :code:`d`: :class:`Timedelta`                  | The duration scaled by the factor :code:`c`, truncated to nanosecond precision            |
+        |                  | * :code:`c`: :class:`float` or :class:`int`      | See :meth:`__mul__`.                                                                      |
+        +------------------+--------------------------------------------------+-------------------------------------------------------------------------------------------+
+        | :code:`d / c`    | * :code:`d`: :class:`Timedelta`                  | The duration divided by :code:`c`, truncated to nanosecond precision.                     |
+        |                  | * :code:`c`: :class:`float`                      | See :meth:`__truediv__`.                                                                  |
```

Comment:
Technically, that's a slight inconsistency to the `d * c` documentation which specifies *float or int*.

Reply:
Well yes... But I really want people to use `//` when they known they want to divide by an integer.

That table is more of a quick reference, the method docs go into more detail.